showcase-app/test-cases.md

```diff
@@ -95,3 +95,32 @@ If you want to run the tests interactively, you use `npm run test:ui`.
   - Reload the page.
   - Verify that the active profile is still displayed.
 
+### 11. Login without extension using WebID
+
+- **Description**: Test the login flow using a WebID input field.
+- **Test Steps**:
+  - Register a profile.
+  - Load main page.
+  - Enter the profile's WebID in the input field, just like a user would.
+  - Log in using profile's credentials.
+  - Verify that upon redirect to the main page, the app is authenticated.
+
+### 12. Displays logged in profile's WebID when available
+
+- **Description**: Test if the WebID's name is displayed when it's returned from the identity provider.
+- **Test Steps**:
+  - Register a profile.
+  - Mock the IDP's response to include a WebID's name.
```

Comment:
This is incorrect because the IDP doesn't return the name.

Reply:
How would you phrase it?

Reply:
We discussed this before. The name is coming from fetching the WebID. The IDP has nothing to do with this.